doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -1171,6 +1171,10 @@ \subsubsection{Inputs}\label{inputs-5-022}
 
 The length in meters of each parallel hydronic circuit in a surface. Used when the previous input field is set to \emph{CalculateFromCircuitLength}. The default is 106.7 meters (350 feet), which is the maximum circuit length allowed in Title 24.
 
+\paragraph{Field: Changeover Delay Time Period Schedule}\label{field-changeover-delay-time-period-schedule}
+
+This field defines a schedule which should be populated with values for the amount of time in hours that is required before this low temperature radiant system can switch between either heating or cooling and the opposite conditioning mode.  If this schedule value is set to 1 (hour) at a particular time and the system was previously in heating mode, then it will be locked out from switching into cooling mode until at least one hour of time has passed in the simulation with the system "not operating".  This helps avoid systems bouncing back and forth between the two conditioning modes in successive time steps.  This parameter can be set to any positive value, and a zero or negative value will be assumed to have no delay (equivalent to a delay of zero).  If the user leaves this field blank, EnergyPlus will assume that no changeover delay is requested (also equivalent to a delay of zero).
```

Comment:
Ideally we would use the native LaTeX quotations: two back ticks to start the phrase, and then two apostrophes to close it:  ` ``my quote'' `.  It's not a critical thing here unless other things need to change.

Reply:
Oh snap.  Mike Witte called this out in a previous PR and since this was done before that other PR got reviewed...  If I need to fix something else, I'll snag this along the way.